<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QBEditWidget</class>
 <widget class="QWidget" name="QBEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_del">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="tre_bases">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>301</height>
    </rect>
   </property>
   <property name="allColumnsShowFocus">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <column>
    <property name="text">
     <string>Base name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Base code</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
